.chloggen/codeboten_fix-missed-logs.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: opampsupervisor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: fix missed logs signal
```

This changelog entry doesn't feel easy to parse/understand.
```suggestion
note: Use OwnLogsConnSettings along with Metrics & Traces settings to evaluate if configuration changed on message.
```